src/gatsby/src/components/park/contact.js

```diff
@@ -81,7 +81,6 @@ export default function Contact({ contact, parkContacts, operations }) {
     operations.hasPicnicShelterReservations ||
     operations.hasFrontcountryReservations ||
     operations.hasFrontcountryCabinReservations ||
-    operations.hasBackcountryPermits ||
     operations.hasBackcountryReservations ||
     operations.hasBackcountryGroupReservations ||
     operations.hasBackcountryShelterReservations ||
```

Comment:
We should also include hasFrontcountryGroupReservations (in place of hasGroupReservations which is noted in the ticket)